AK/SIMDMath.h

```diff
@@ -38,6 +38,18 @@ ALWAYS_INLINE static f32x4 frac_int_range(f32x4 v)
     return v - floor_int_range(v);
 }
 
+template<class T>
+ALWAYS_INLINE T bitselect(T v1, T v2, T control_mask)
+{
+    return (v1 & control_mask) | (v2 & ~control_mask);
+}
+
+template<class T>
+ALWAYS_INLINE T abs(T x)
+{
+    return bitselect(x, -x, x > 0);
```

Comment:
Nit: This does not work for floating point types

Reply:
Do we have traits for "is integral simd vector"?

At least it does not work in the "it does not compile when you try to use it" sense, not in the "it compiles but does the wrong thing" sense, so we could worry about the float version when we need it.

Reply:
Added a `requires(IsIntegral<ElementOf<T>>)` to this.